### DISCUSSION TITLE
Pick Up Objects Using A Robot Hand

### DISCUSSION THREAD

**Discussion by bkhan23544** [enable3d/enable3d#197] [Help / Questions] [answered]
I am trying to pick objects using a robot hand. I have used just 2 cubes to just experiment. But it seems like it's not picking the cube up. I tried adding friction to objects too. But it doesn't work.
Check the gif.
[https://imgur.com/d5aSVHI](https://imgur.com/d5aSVHI)

Is there any solution for this?

**Comment by yandeu** [comment, marked as answer]
Try increasing the mass of the arms.
Are your arms kinematic?

**Reply by bkhan23544** [reply]
Yes, the arms are kinematic. I tried increasing mass, but it still doesn't pick it up.

**Reply by yandeu** [reply]
I have just added an example to the website.
https://enable3d.io/examples/standalone/pick-up-objects.html

**Reply by bkhan23544** [reply]
Thank you very much!

**Reply by bkhan23544** [reply]
So, it just works if the hands are just touching the cube, if hands just go a little bit inside the cube, it doesn't pick up. I was using gsap to move them

**Reply by yandeu** [reply]
I don't know what gsap is, but if you are looking for a better solution to this problem, try the [bullet forum](https://pybullet.org/Bullet/phpBB3/).
